src/DataGrid.tsx

```diff
@@ -880,12 +880,14 @@ function DataGrid<R, SR>({
       const top = rowIdx * rowHeight + totalHeaderHeight;
       if (isGroupRow(row)) {
         ({ startRowIndex } = row);
+        const isGroupRowSelected = isSelectable && row.childRows.every(cr => selectedRows?.has(rowKeyGetter!(cr)));
         rowElements.push(
           <GroupRowRenderer<R, SR>
             aria-level={row.level + 1} // aria-level is 1-based
             aria-setsize={row.setSize}
             aria-posinset={row.posInSet + 1} // aria-posinset is 1-based
             aria-rowindex={headerRowsCount + startRowIndex + 1} // aria-rowindex is 1 based
+            aria-selected={isSelectable ? isGroupRowSelected : undefined}
```

Comment:
This will also avoid `aria-selected="false"`
```suggestion
            aria-selected={isGroupRowSelected || undefined}
```

Reply:
> If the grid supports selection, when a cell or row is selected, the selected element has aria-selected set true. If the grid supports column selection and a column is selected, all cells in the column have aria-selected set to true.
https://www.w3.org/TR/wai-aria-practices/#grid_roles_states_props

From what I understand it should only be set if `isSelectable ` is true

Reply:
Yes that's in agreement with my suggestion.